pytarallo/ProductToUpload.py

```diff
@@ -0,0 +1,9 @@
+from pytarallo.Product import Product
```

Comment:
We are in the `pytarallo` directory/module already, so this should work:
```suggestion
from Product import Product
```